x-pack/plugins/cloud_security_posture/common/constants.ts

```diff
@@ -38,8 +38,6 @@ export const DETECTION_RULE_RULES_API_CURRENT_VERSION = '2023-10-31';
 export const CLOUD_SECURITY_POSTURE_PACKAGE_NAME = 'cloud_security_posture';
 
 export const CDR_MISCONFIGURATIONS_DATA_VIEW_NAME = 'Latest Cloud Security Misconfigurations';
```

Comment:
even if we don't need this const for shared code, I'd move it too to avoid confusion, as CDR related constants belong together

Reply:
good call